app/models/agents/webhook_agent.rb

```diff
@@ -20,6 +20,9 @@ class WebhookAgent < Agent
         * `payload_path` - JSONPath of the attribute in the POST body to be
           used as the Event payload.  If `payload_path` points to an array,
           Events will be created for each element.
+        * `verbs` - Comma-separated list of http verbs your agent will accept.
```

Comment:
Ahh... yeah, I wasn't thinking about GET in that fashion, but hmm... I agree with that.  I'm not some REST fanatic, but I agree with sticking to those basic conventions as tight as possible.

My biggest problem with allowing GET by default is that the agent has a link to the URL to use.  But if you click that link, you'll fire an event.  

Anyway, I'll make the change soon, just give me a few hours.